app/controllers/users_controller.rb

```diff
@@ -1,7 +1,7 @@
 class UsersController < ApplicationController
   before_action :set_user, only: [:show, :edit, :update, :destroy, :impersonate]
 
-  load_and_authorize_resource except: [:index, :show, :impersonate, :stop_impersonating]
+  load_and_authorize_resource except: [:index, :show, :impersonate, :stop_impersonating, :set_available]
```

Comment:
Since this list is getting quite long now, why not "reverse" it and use `only` instead?
```ruby
load_and_authorize_resource only: %i[new create edit update destroy]
```